src/js/components/navbars.js

```diff
@@ -85,48 +86,127 @@
 
   var module = angular.module('mobile-angular-ui.components.navbars', []);
 
-  /**
-   * @directive navbarAbsoluteTop
-   * @restrict C
-   * @description
-   *
-   * Setup absolute positioned top navbar.
-   *
-   * ``` html
-   *  <div class="navbar navbar-app navbar-absolute-top">
-   *    <!-- ... -->
-   *  </div>
-   * ```
-   */
+  var global;
 
   /**
-   * @directive navbarAbsoluteBottom
-   * @restrict C
-   * @description
-   *
-   * Setup absolute positioned bottom navbar.
-   *
-   * ``` html
-   *  <div class="navbar navbar-app navbar-absolute-bottom">
-   *    <!-- ... -->
-   *  </div>
-   * ```
-   */
+  * @directive navController
+  * @restrict C
+  * @description
+  *
+  * Directive to bind controller to navbar
+  *
+  * ``` html
+  *  <div class='navbar navbar-app navbar-absolute-top' nav-controller>
+  *    <!-- ... -->
+  *  </div>
+  * ```
+  */
+  module.directive('navController', function() {
+    return {
+      restrict: 'A',
+      controller: function() {
+        this.registerController = function(name, ctrl) {
+          this[name] = ctrl;
+        };
+      },
+      link: function(scope) {}
+    };
+  });
+
+ /**
+  * @directive navbarAbsoluteTop
+  * @restrict C
+  * @description
+  *
+  * Setup absolute positioned top navbar.
+  *
+  * ``` html
+  *  <div class='navbar navbar-app navbar-absolute-top'>
+  *    <!-- ... -->
+  *  </div>
+  * ```
+  */
+
+ /**
+  * @directive navbarAbsoluteBottom
+  * @restrict C
+  * @description
+  *
+  * Setup absolute positioned bottom navbar.
+  *
+  * ``` html
+  *  <div class='navbar navbar-app navbar-absolute-bottom'>
+  *    <!-- ... -->
+  *  </div>
+  * ```
+  */
   angular.forEach(['top', 'bottom'], function(side) {
     var directiveName = 'navbarAbsolute' + side.charAt(0).toUpperCase() + side.slice(1);
     module.directive(directiveName, [
       '$rootElement',
       function($rootElement) {
         return {
           restrict: 'C',
-          link: function(scope) {
+          require: 'navController',
```

Comment:
You could have used the same directive, there is no problem in extending it, as soon as its behaviour remains backward compatible.